Source/AwsCommonRuntimeKit/sdkutils/endpoint/EndpointsRequestContext.swift

```diff
@@ -48,6 +48,34 @@ public class EndpointsRequestContext {
         }
     }
 
+    private func withByteCursorArrayFromStringArray<R>(
```

Comment:
Trivial: We should move this to Utilities.swift, as this will probably get reused in the future. We should also add a test case for just this function.